README.md

```diff
@@ -0,0 +1,43 @@
+# Babelfish Tools
+
+[![Build Status](https://travis-ci.org/bblfsh/tools.svg?branch=master)](https://travis-ci.org/bblfsh/tools)
+[![codecov](https://codecov.io/gh/bblfsh/tools/branch/master/graph/badge.svg)](https://codecov.io/gh/bblfsh/tools)
+
+Language analysis tools on top of Babelfish
+
+## Build
+
+### With docker
+
+`make build`
+
+### Without docker
+
+`make build-internal`
+
+## Usage
+
+Babelfish Tools provides a set of tools built on top of Babelfish, to
+see which tools are supported, run:
+
+`bblfsh-tools --help`
```

Comment:
I guess Go users might just use the binary. Maybe we can put the command now, and once the container is published to Docker Hub, we can also add the docker command.